assembl/static2/js/app/utils/linkify.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import linkifyHtml from 'linkifyjs/html';
+import * as linkify from 'linkifyjs';
+
+export function transformLinksInHtml(html: string): string {
+  const linksToReplace = linkify
+    .find(html)
+    .map((link) => {
+      const url = link.href;
+      if (url.startsWith('https://www.youtube.com') || url.startsWith('https://youtube.com')) {
+        const re = /(.*watch\?v=(.*?))([\s<,;].*)/;
```

Comment:
Could you explain this group further to me? `([\s<,;].*)`?

Reply:
it ensures that the regexp ends if there is a space, a comma or a `<` (because sometimes, `linkify.find` returns "bad" urls like : `http://www.example.com/</p>`)